frontend/webapp/components/side.menu/menu/menu.tsx

```diff
@@ -76,15 +78,14 @@ export function Menu() {
   function renderContactUsButton() {
     return (
       <ContactUsWrapper>
-        <ContactUsButton expand={isHovered} />
+        <ContactUsButton expand={isExpanded} />
       </ContactUsWrapper>
     );
   }
 
   return (
     <MenuContainer
-      onMouseEnter={() => setIsHovered(true)}
-      onMouseLeave={() => setIsHovered(false)}
+      className={isExpanded ? 'expanded' : ''}
```

Comment:
@HeetVekariya please send isExpanded as styled-component props to the width attribute to match the rest of the app

Reply:
Changes made